### PR TITLE
Docs: Add quick start guides for dragonfly cloud

### PR DESCRIPTION
**Pull request by s1ntaxe770r**
This PR adds docs for:

 - getting started guide for data store
 - Security and network features of dragonfly cloud

### CODE REVIEW

File: docs/cloud/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+1. From the **Data Stores** menu, at the top right corner of the dashboard,  click **Data Store**
+
+![dashboard](../../static/img/dashboard.png)
+
+
+2. Fill out the "Create Data Store" form:
+
+![create-datastore](../../static/img/create-datastore.png)
+
+   - **Name:** Provide a unique name for your data store.
```
Comment (ashotland) [inline docs/cloud/getting-started.md:19]: we don't enforce uniqueness

File: docs/cloud/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+   - **Cloud Provider:**
+     - **AWS:** Select this option to create your data store on Amazon Web Services.
+     - **GCP:** Select this option to create your data store on Google Cloud Platform.
+   - **Region:**
+     - For AWS, you can choose from `us-east-1` and `eu-west-1`. More regions are coming soon.
```
Comment (ashotland) [inline docs/cloud/getting-started.md:24]: those regions happens to be the one staging, prod has many more, I don't think we should specify them here

File: docs/cloud/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+---
+sidebar_position: 1
+title: Dragonfly Quickstart
```
Comment (ashotland) [inline docs/cloud/getting-started.md:3]: my intention for this page to cover onboarding and signup and very basic datastore then link to the datastore pages for more details

File: docs/cloud/networks.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
+---
+sidebar_postion: 2
+--- 
+
+
+## Managing Networks on Drangonfly Cloud
+
+Dragonfly Cloud allows you to create isolated networks where you can deploy your data stores similar to VPC's if you are familiar with AWS.
```
Comment (ashotland) [inline docs/cloud/networks.md:8]: the motivation for a private network is security, latency and cost (as you avoid egress charges) 

it should proceed to connecting to the private network on the  "peering connection" page

File: docs/cloud/getting-started.md (hunk 3 above)
Comment (s1ntaxe770r) [reply docs/cloud/getting-started.md:3]: will update this in the next pr, along with the page structure we went over

File: docs/cloud/cloud.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
+---
+sidebar_position: 1
+label: Cloud
+--- 
+
```
Comment (ashotland) [inline docs/cloud/cloud.md:5]: this is the content of the data stores page

File: docs/cloud/cloud.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+## Getting Started with Dragonfly Cloud
+
+A Dragonfly Cloud data store represents a Redis Protocol (RESP) endpoint. To create a data store, on the Data Store tab click +Data Store. The minimum configuration consists of a name, cloud provider, cloud region and plan. Note that the cloud provider can not be modified once the data store is created.
```
Comment (ashotland) [inline docs/cloud/cloud.md:9]: please format better with line breaks please

Comment (ashotland) [reply docs/cloud/cloud.md:9]: Can you use an image for +Data Store 
image

File: docs/cloud/cloud.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+![datastore](../../static/img/create-datastore.png)
+
+
+The plan specifies the provisioned memory and memory to cpu ratio of the data store. You can modify the data store plan later with zero down time to scale it up or down.
```
Comment (ashotland) [inline docs/cloud/cloud.md:14]: new lint for 2nd sentence

File: docs/cloud/cloud.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+
+By default the data store will be configured with a public endpoint, TLS and an auto generated passkey, meaning you can securely connect to it from anywhere over the public internet.
+
+To create a data store with a private endpoint, see [Security](cloud/data-stores/security), [Networks](cloud/networks) and Peering Connections.By default the data store will consist of a single Dragonfly server, to create a highly available data store see [Durability and High Availability](cloud/data-stores/duarability-vs-high-availability)
```
Comment (ashotland) [inline docs/cloud/cloud.md:18]: Space after Connections.

Comment (ashotland) [reply docs/cloud/cloud.md:18]: Peering Connection should link

File: docs/cloud/cloud.md [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+
+Once the data store is created, clicking the data store row will open a drawer with the data store configuration, including the auto generated passkey and redis compatible connection string. 
+
+
+![edit datastore](../../static/img/datastore-edit.png)
```
Comment (ashotland) [inline docs/cloud/cloud.md:23]: The image here should show the drawer with the details when clicking a data store row

File: docs/cloud/data-stores/create-datastore.md [hunk 10]
```diff
@@ -0,0 +1,40 @@
+---
+sidebar_position: 1
+title: Creating a Data Store
```
Comment (ashotland) [inline docs/cloud/data-stores/create-datastore.md:3]: please remove this content

File: docs/cloud/data-stores/duarability-vs-high-availability.md [hunk 11]
```diff
@@ -0,0 +1,32 @@
+---
+sidebar_position: 3
+title: Durability and Availability
+--- 
+
+
+# Durability and Availability
+Dragonfly Cloud prioritizes both data durability and high availability for your data stores. You can further fine-tune these aspects through the concept of eviction policies when creating a data store. This allows you to strike a balance between data persistence and performance based on your application's needs.
```
Comment (ashotland) [inline docs/cloud/data-stores/duarability-vs-high-availability.md:8]: please stick to the content I specified in
https://docs.google.com/document/d/1kipawV9069mc8iSZEP6OuiVsGlQkS6JTs5eZEQAYjMo/edit

Comment (ashotland) [reply docs/cloud/data-stores/duarability-vs-high-availability.md:8]: make sure to do so across the entire PR

**Review by ashotland — CHANGES_REQUESTED** [review]

File: docs/cloud/networks.md [hunk 12]
```diff
@@ -0,0 +1,49 @@
...
+- Create a data store in the private network with a private endpoint For more details see [Data Store Security](./data-stores/security).
+
+
+For more details see [Peering Connections](./peering). Create a data store in the private network with a private endpoint For more details see Data Store Security
+
+
+
+A private endpoint data store in a private network provides better security, performance and reduces data transfer costs.
+
+To create a private network navigate to the Networks tab and click +Network at the top right corner.
```
Comment (ashotland) [inline docs/cloud/networks.md:22]: Can you use an image dor +Network ?

File: docs/cloud/data-stores/quick-start.md [hunk 13]
```diff
@@ -0,0 +1,39 @@
...
+
+
+## Getting Started with Dragonfly Cloud
+
+A Dragonfly Cloud data store represents a Redis Protocol (RESP) endpoint. To create a data store, on the Data Store tab click +Data Store. 
+
+The minimum configuration consists of a name, cloud provider, cloud region and plan. Note that the cloud provider can not be modified once the data store is created.
+
+
+![datastore](../../../static/img/add-datastore.png)
```
Comment (ashotland) [inline docs/cloud/data-stores/quick-start.md:15]: This doesn't look good. Can you make it smaller and inline with the text?

**Review by ashotland — CHANGES_REQUESTED** [review]
Still doesn't match the organization of doc I've sent.

File: docs/cloud/data-stores/quick-start.md (hunk 13 above)
Comment (s1ntaxe770r) [reply docs/cloud/data-stores/quick-start.md]: should be fixed now

Comment (s1ntaxe770r) [reply docs/cloud/data-stores/quick-start.md]: docs should line up now

**Pull request closed without merging**